doc/release_notes/v2.4.0.rst

```diff
@@ -0,0 +1,9 @@
+Version 2.4.0
+=================================
+
+Enhancements
+------------
+* Added :meth:`~pydicom.valuerep.PersonName.alphabetic` enum (:pr:`1634`)
+
```

Comment:
Small thing, but it's not an enum, rather a property.

Reply:
done!

Reply:
You still name it an enum...

Reply:
ugh, sorry about that, changed the hyperlink to attr: like the docs https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects but forgot that. Do you want me to submit a PR to change it or maybe another contribution should just fix it?

Reply:
No, thanks - that's just a minor glitch we can fix within another PR.